src/main/java/tech/gruppone/stalker/server/repositories/PlacePositionRepository.java

```diff
@@ -10,11 +11,18 @@
 public interface PlacePositionRepository extends ReactiveCrudRepository<PlacePositionDao, Long> {
 
   @Query("SELECT id, ST_ASGEOJSON(position) AS positionGeoJson FROM PlacePosition WHERE id = :id")
-  Mono<PlacePositionDao> findById(final Long id);
+  Mono<PlacePositionDao> findById(@Param("id") final Long id);
 
   // returns number of affected rows (should be 0 or 1)
   @Modifying
   @Query(
       "INSERT INTO `PlacePosition`(`id`, `position`) VALUES (:id, ST_GeomFromGeoJSON(:rawPositionJson))")
-  Mono<Integer> save(final Long id, final String rawPositionJson);
+  Mono<Integer> save(
+      @Param("id") final Long id, @Param("rawPositionJson") final String rawPositionJson);
+
+  @Modifying
+  @Query(
+      "UPDATE PlacePosition SET `position` = ST_GeomFromGeoJSON(:rawPositionJson) WHERE `id` = :id")
```

Comment:
E' stato provato il funzionamento di questo metodo senza la scrittura esplicita della query?

Reply:
in questo caso non è possibile evitarlo perché dobbiamo chiamare una funzione di mysql ( ST_GeomFromGeoJSON) per convertire le coordinate a WKB